C++/main.cpp

```diff
@@ -0,0 +1,27 @@
+#include <QCoreApplication>
+#include <iostream>
+#include <linkedlist.h>
+
+using namespace std;
+
+int main()
+{
+
+    LinkedList<int> *list = new LinkedList<int>();
```

Comment:
why not just 

LinkedList<int> list; 
list.add(1);

Why did you have to use pointer in creating this object?